cmd/flowrunner/testdata/flows/all_actions.json

```diff
@@ -56,7 +56,7 @@
                             "type": "send_email",
                             "subject": "Here is your activation token",
                             "body": "Hi @contact.fields.first_name, Your activation token is @contact.fields.activation_token, your coupon is @run.extra.coupons.0.code",
-                            "emails": [
+                            "addresses": [
```

Comment:
think this is clearer

Reply:
I don't care much, but why do you think it is clearer? `emails` seems ambiguous while `addresses` isn't.

Reply:
We agree `emails` is ambiguous - this change is from `emails` to `addresses `

Reply:
Maybe you should reread that. :)

Reply:
oops - oh then we don't agree. `email` is ambiguous - could mean the body or an address

Reply:
oh nice try editing that comment!

Reply:
I edited before you commented. :)

What is your email is ambiguous to you? But, whatever. Godspeed.

Reply:
but not before I read your comment in the **EMAIL** from github.

In the question "what is your email" it's obvious you mean "email address". In the sentence "he sent an email", it's obvious that we're talking about the entire message. On it's own, it's ambiguous.